Week-3/day6/recipe/styles.css

```diff
@@ -0,0 +1,80 @@
+/* Add your styles here */
+
+/* Given CSS rules below */
+body {
```

Comment:
There are no specific issues with the provided CSS, but as a general practice, consider using more descriptive class names over generic ones, such as .main-content, .title, .summary, etc.